lib/utils.js

```diff
@@ -100,6 +100,14 @@ class FolderPath {
     );
   }
 
+  pathWithoutSeperator(){
+    if (this.isEmpty()) return new RawParam();
+
+    return new RawParam(
+      "/" + this.value.map(encodeURIComponent)
```

Comment:
should this be `"/" + this.value.map(encodeURIComponent).join("/")`, otherwise you're basically doing something like `"/" + ["test"]`.

Reply:
Other option is just to update path to support an optional `sep`:

```
  path(sep) {
    if (!sep) sep = this.SEP;
    if (this.isEmpty()) return new RawParam();
    return new RawParam(
      sep + this.value.map(encodeURIComponent).join(sep)
    );
  }
```